vm/instruction.go

```diff
@@ -283,7 +283,7 @@ var builtInActions = map[operationType]*action{
 				constant := vm.constants[constantName]
 				inheritedClass, ok := constant.Target.(*RClass)
 				if !ok {
-					newError("Constant %s is not a class. got=%T", constantName, constant)
+					panic("Constant "+ constantName +" is not a class. got=" + string(constant.Target.objectType()))
```

Comment:
@janczer Because `newError` returns an error object, but we didn't return it here so it didn't work at all.  But I will change it to pushing error to the stack.